net/net.sh

```diff
@@ -167,6 +167,7 @@ numClients=${#clientIpList[@]}
 numClientsRequested=$((numBenchTpsClients+numBenchExchangeClients))
 if [[ "$numClientsRequested" -eq 0 ]]; then
   numBenchTpsClients=$numClients
+  numClientsRequested=$((numBenchTpsClients+numBenchExchangeClients))
```

Comment:
how about moving `numClientsRequested=$((numBenchTpsClients+numBenchExchangeClients))` to after the if/else block (right before `build()`), then remove this line and the duplicate line that's 3 lines above

Reply:
err wait that doesn't quite work :)

Reply:
```bash
numClients=${#clientIpList[@]}
numClientsRequested=$((numBenchTpsClients+numBenchExchangeClients))
if [[ "$numClientsRequested" -eq 0 ]]; then
  numBenchTpsClients=$numClients
  numClientsRequested=$numBenchTpsClients
fi
if [[ "$numClientsRequested" -gt "$numClients" ]]; then
  echo "Error: More clients requested ($numClientsRequested) then available ($numClients)"
  exit 1
fi

```

Reply:
Line 168 is checking $numClientsRequested, so it can't be moved after the if block.

Reply:
In the above checks, if numClientsRequested is 0, it can never be > numClients.
as, numClientsRequested = numBenchTpsClients = numClients

Reply:
@mvines do you have any further comments?